frontend/src/app/core/helpers/time-helper.ts

```diff
@@ -1,6 +1,35 @@
 import { AbstractControl, ValidationErrors, ValidatorFn } from '@angular/forms';
+import { DayAction } from '@core/enums/day-action.enum';
 
-export function TimeRangeValidator(firstControl: AbstractControl): ValidatorFn {
-    return (control: AbstractControl): ValidationErrors | null =>
-        (firstControl?.value < control?.value ? null : { error: true });
+export const TimeRangeValidator =
+    (firstControl: AbstractControl): ValidatorFn =>
+        (control: AbstractControl): ValidationErrors | null =>
+            (firstControl?.value < control?.value ? null : { error: true });
+
+export function normalizeHours(hours: number): [number, DayAction] {
+    if (hours < 0) {
+        return [24 + hours, DayAction.SubtractDay];
+    }
+    if (hours >= 24) {
+        return [hours - 24, DayAction.AddDay];
+    }
+
+    return [hours, DayAction.NoAction];
+}
+
+export const getTimeString = (hours: number, minutes: string) =>
+    `${hours.toString().length < 2 ? '0' : ''}${hours}:${minutes}`;
```

Comment:
Why do we need this?

Reply:
We need this in order to have always string in format "00:00" with four digits and ':' symbol

Reply:
Fixed